core/commonMain/src/ArgParser.kt

```diff
@@ -132,7 +132,7 @@ open class ArgParser(
     /**
      * Map of subcommands.
      */
-    @UseExperimental(ExperimentalCli::class)
+    @OptIn(ExperimentalCli::class)
```

Comment:
Better to extract UseExperimental->OptIn replacement to a separate commit.

Reply:
Fixed